src/app/state/accountHistorySlice.ts

```diff
@@ -53,6 +55,7 @@ export const fetchAccountHistory = createAsyncThunk<
   }
 });
 
+// this is just a normal async function and should not be a thunk.
```

Comment:
This function currently uses `thunkAPI.dispatch` and `thunkAPI.getState()` so it looks like its interacting with state. I'd need again an explenation why it should not be a thunk.

Reply:
Yep, you're right again. The fact that it uses state does not make it a thunk. Any component/code can use state. But, the fact that it dispatches an action as a result of something else happening (the transaction has been submitted) does make it a valid thunk.